app/views/groups/edit_invitations.html.erb

```diff
@@ -1,3 +1,6 @@
+<%= provide(:title, @group.name + " - Edit invitations") %>
+<h1> Edit invitations</h1>
```

Comment:
Extra space at the start of the tag (and the next one) - also, the group name should probably be the `h1` rather than the activity.